Message: Interface 'MessageProps' incorrectly extends interface ...

### Describe the bug

When I typecheck my code, I get a type error in message.d.ts:

```
node_modules/primereact/message/message.d.ts:17:18 - error TS2430: Interface 'MessageProps' incorrectly extends interface 'Omit<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, "ref">'.
  Types of property 'content' are incompatible.
    Type 'ReactNode | ((props: MessageProps) => ReactNode)' is not assignable to type 'string | undefined'.
      Type 'null' is not assignable to type 'string | undefined'.

17 export interface MessageProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref'> {
```

### Reproducer

_No response_

### PrimeReact version

9.2.3

### React version

18.x

### Language

TypeScript

### Build / Runtime

Vite

### Browser(s)

Chrome 111.0.5563.149

### Steps to reproduce the behavior

Open message.d.ts with linting enabled

### Expected behavior

_No response_

--- a/components/lib/message/message.d.ts
+++ b/components/lib/message/message.d.ts
@@ -14,7 +14,7 @@ import { IconType } from '../utils/utils';
  * Defines valid properties in Message component. In addition to these, all properties of HTMLDivElement can be used in this component.
  * @group Properties
  */
-export interface MessageProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref'> {
+export interface MessageProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref' | 'content'> {
     /**
      * Text of the message.
      */

